### PR TITLE
docs: fix indentation in contributors list

### PR DESCRIPTION
**Pull request by blove**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Corrected indentation in the _aio/content/marketing/contributors.json_ file. CC @gkalpak 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -689,5 +689,14 @@
...
     "group": "GDE"
+  },
+
+  "blove": {
+    "name": "Brian Love",
+    "picture": "brian-love.jpg",
+    "twitter": "brian_love",
+    "website": "https://brianflove.com",
+    "bio": "Brian is a software engineer and GDE in Angular with a passion for learning, writing, speaking, teaching and mentoring. Brian has been building web applications for over 20 years and has long been a fanboy of JavaScript. When not in front of his Macbook Pro Brian is in the Rocky Mountains skiing or hiking.",
+    "group": "GDE"
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: Remove this change from this commit :grin:
This one should be only about indentation.

**Review by gkalpak — CHANGES_REQUESTED** [review]

**Comment by blove** [comment]
@gkalpak I've rebased from master and fixed the indention accordingly.

**Review by gkalpak — APPROVED** [review]

**Comment by jenniferfell** [comment]
@blove : Is this just code cleanup? I don't see any visible user impact. #curious

**Comment by blove** [comment]
@jenniferfell yes, that's correct. This simply fixes the indentation as per #26594

**Pull request closed without merging**